ext/fetch/23_response.js

```diff
@@ -499,9 +495,11 @@ function toInnerResponse(response) {
  * @returns {Response}
  */
 function fromInnerResponse(inner, guard) {
-  const response = new Response(_brand);
+  const response = {};
+  ObjectSetPrototypeOf(response, Response.prototype);
   response[_response] = inner;
   response[_headers] = headersFromHeaderList(inner.headerList, guard);
+  response[_brand] = _brand;
```

Comment:
Also makes the order of this object match what's done in the ctor, which I think should have with monomorphism.